ansible_collections/arista/avd/plugins/action/eos_validate_state_runner.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2023 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+import logging
+
+from ansible.errors import AnsibleActionFail
+from ansible.module_utils.common.arg_spec import ArgumentSpecValidator, ValidationResult
+from ansible.module_utils.errors import UnsupportedError
+from ansible.plugins.action import ActionBase, display
+
+from ansible_collections.arista.avd.plugins.plugin_utils.eos_validate_state_utils import AnsibleEOSDevice, get_anta_results
+from ansible_collections.arista.avd.plugins.plugin_utils.utils import PythonToAnsibleContextFilter, PythonToAnsibleHandler
+
+LOGGER = logging.getLogger("ansible_collections.arista.avd")
+# ANTA 0.8.0 currently add some RichHandler to the root logger so need to disable propagation
+LOGGER.propagate = False
+
+
+class ActionModule(ActionBase):
+    def run(self, tmp=None, task_vars=None):
+        self._supports_check_mode = True
+
+        if task_vars is None:
+            task_vars = {}
+
+        validation_result, new_module_args = self._validate_argument_spec()
```

Comment:
You are not checking validation_result. So issues are ignored...

Reply:
dropped!